src/screens/HomeScreen.js

```diff
@@ -1,11 +1,13 @@
 import React from 'react';
-import { Text, View } from 'react-native';
+import { Image } from 'react-native';
+
 import styles from 'src/styles/homeScreenStyle';
+import images from '../assets/images';
+
+const { homeBgImage } = images.home;
 
 const HomeScreen = () => (
-  <View style={styles.container}>
-    <Text>Hello World</Text>
-  </View>
+  <Image source={homeBgImage} style={styles.imageBackground} resizeMode="cover" />
```

Comment:
add this component in View component because HomeScreen component will contain other components

Reply:
Great.